_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -466,7 +479,7 @@ This link is not distinguishable by color (hue) from the other text rendered in
 [flat tree]: https://drafts.csswg.org/css-scoping/#flat-tree 'Definition of flat tree'
 [foreground color]: #foreground-colors-of-text 'Definition of foreground colors of text'
 [inclusive descendant]: https://dom.spec.whatwg.org/#concept-tree-inclusive-descendant 'Definition of inclusive descendant'
-[link history states]: https://drafts.csswg.org/selectors-4/#link 'Definition of Link History pseudo-classes'
+[link history state]: https://drafts.csswg.org/selectors-4/#link 'Definition of Link History pseudo-classes'
```

Comment:
We should link to the definitions in https://html.spec.whatwg.org/multipage/semantics-other.html#pseudo-classes
which might require an intermediate definition 😕

Reply:
I considered this when preparing the last PR, but I could not convince myself about the advantages of doing it. 

- There is no way, that I could think of, to reflect the semantics of :link and :visited from the html spec, while these are immediately understandable from the css spec. 
- And, as you say, replacing this would require an indirection (the intermediate definition, that in fact would be doing exactly the same that the link to the css spec does).

Could you elaborate on why we should link to the html spec?

Reply:
Indeed I haven't looked into too much detail into the HTML version, but for `:link` and `:visited`, it is not adding much…

OTOH, it is adding a lot of precision for `:hover` and `:focus`… And we already have a definition for "inspection states". Maybe there we could update the link, or at least add a note "for HTML elements, see also …"

Reply:
That might be more useful, indeed. I'll have another look at this.